src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java

```diff
@@ -191,6 +192,7 @@
             + "|\\\\n"
             + "|\\\\r"
             + "|\\\\t"
+            + "|\\\\s"
```

Comment:
1. Please change the link to https://docs.oracle.com/javase/specs/jls/se15/html/jls-3.html#jls-3.10.7

2. All patterns in this file seems to be sorted. It is better to place `\s` between `\r` and `\t`

3. The docs says there is one more escape - for a trailing `\`. Something like `\\$`.
For example
```java
    static final String value1 = """
        \u03bc\
        \s\u03bc\
        """; // ok
```
is just `μ μ`. Please create a new issue for this.

Reply:
1. Done
2. Done.
3. https://github.com/checkstyle/checkstyle/issues/9143